Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -5617,4 +5620,11 @@ LayoutUnit synthesizedBaselineFromBorderBox(const RenderBox& box, LineDirectionM
     return direction == HorizontalLine ? box.height() : box.width();
 }
 
+bool RenderBox::shouldIgnoreMinMaxSizes() const
+{
+    if (RenderElement *flexContainer = parent(); flexContainer && isFlexItem()) 
```

Comment:
Seems like here testing isFlexItem() is enough since it implies the parent existing and being a flex container.

Reply:
Good catch! Should be changed now. Since we still need the parent inside the if clause there's still a call the parent(), but this should reduce the number of calls when we don't ignore min-max sizes.